### ISSUE TITLE
Idea is good. But image somehow always fail at loading 

### ISSUE THREAD

**Issue by nguyenbh295** [hanFengSan/eHunter#32]
Nice idead by the way. Kudos. But somehow I have some problem at loading image. It always fail. But seem like original image is still work. It there any way to always load original image ?

**Comment by hanFengSan** [comment]
Can it work when you close the eHunter? There don't have a way to always load original image. If it still have same problem after closing eHunter, I think that it is because of your network.

**Comment by nguyenbh295** [comment]
no. It still work when eHunter close. That why the original future still work for me but i have to click "Original" in every image tho.

**Comment by nguyenbh295** [comment]
Example : ![image](https://user-images.githubusercontent.com/35690999/41221907-bb8b1bac-6d8f-11e8-8af1-0a2bc0444317.png)

When I clicked "Original" : 
![image](https://user-images.githubusercontent.com/35690999/41221987-fdc8d7ca-6d8f-11e8-9ce8-a6d0f463aa39.png)

When Ehunter is turned off: 

![image](https://user-images.githubusercontent.com/35690999/41222017-13be1f2c-6d90-11e8-8494-b96c15146066.png)

**Comment by hanFengSan** [comment]
Do you try the 'Other source'? Can it work? You can press `F12`  on Windows or `command+option+i` on Mac, and choose the tab of `Console` on the top, send the screenshots to me. I maybe can find out the true problem.

**Comment by nguyenbh295** [comment]
Here is the screenshot: 
![image](https://user-images.githubusercontent.com/35690999/41235200-70be1c68-6db8-11e8-8fc0-2aba55e69a78.png)

**Comment by hanFengSan** [comment]
It don't have any useful information. I think this is because you to get the screenshot too early, can you wait about 1 min and get it again?
I also see the Chrome in your screenshot, If possible, I hope you to get a Chrome version and try it. If the Chrome version still don't work, you can press F12 and select the `Console` to send the screenshot tome.  It will get some very useful informations.
thx :).

**Comment by nguyenbh295** [comment]
Here is Firefox after I scroll to the end. 
![image](https://user-images.githubusercontent.com/35690999/41285692-3a6bfbf8-6e67-11e8-9981-185d85117ebc.png)

Here is Chrome after I scroll to the end. I think Chrome work fine at first, but then I relise images are not loading at the right order. As you can see thumbnails on the left and current page I was in are not the same. Same with all the the other pager. Wrong order. 

![image](https://user-images.githubusercontent.com/35690999/41285879-d411f708-6e67-11e8-862b-6760044c7f28.png)

**Comment by hanFengSan** [comment]
![image](https://user-images.githubusercontent.com/16065654/41291336-04afa7b0-6e82-11e8-8bc6-0ae65ac62a2b.png)
Please check the your number of 'rows', it is a known bug if the number isn't 4 and it will cause the bug of wrong order.

**Comment by nguyenbh295** [comment]
I can only select 4 rows bu defaut. I can't select an other row. Still wrong order. What about Firefox. Do you have any idea how to fix it ?

**Comment by hanFengSan** [comment]
It's very werid, and I don't have any idea how to fix it now. In this weekend, I will try to resolve the  problem again :)

**Comment by nguyenbh295** [comment]
Thank you. No worry. I really like your extension, now I can read comic without downloading it. I'm here to help so if you need any screenshoot, code,.... I willing to provide them :)

**Comment by nguyenbh295** [comment]
Firefox eHunter still don't work :(
